frontend/src/pages/Test/components/ColorPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import CheckSVG from "@/assets/svgs/check.svg?react";
+import { useRecoilValue } from "recoil";
+import { useEffect, useState } from "react";
+import { fabric } from "fabric";
+import cavasInstanceState from "./stateCanvasInstance";
+
+interface ColorPannelInterface {
+  className: string;
+}
+type PenColorTypes = "red" | "yellow" | "forsythia" | "lightGreen" | "blue" | "black";
+
+const colorCode = {
+  red: "#DF5536",
+  yellow: "#F2C947",
+  forsythia: "#FCF467",
+  lightGreen: "#D3E660",
+  blue: "#5099E9",
+  black: "#000000"
+};
+
+const ColorPanel = ({ className }: ColorPannelInterface) => {
```

Comment:
ColorPannelInterface를 인터페이스로 분리한 것이 불필요하게 느껴지는데 아래처럼 바꾸는 건 어떤가요?
```suggestion
const ColorPanel = ( className : srting) => {
```

Reply:
아하 인터페이스 내부에 여러 프로퍼티가 있었는데 지워가는 와중에 하나만 남았는데 별생각 없이 방치했네요 ㅎㅎ
확실히 인터페이스를 사용할 이유가 없어졌습니다 감사합니다.

Reply:
적용해 보던 중 props는 object로 전달되는 것을 확인 했고 ```const ColorPanel = (className: { className: string }) => {``` 이렇게 타입을 주는 것으로 해결했습니다.

Reply:
구조분해 할당을 적용하기 위해 ```{ className }: { className: string }``` 이렇게 바꿨습니다!